### PR TITLE
Profiles service integration

### LINKED ISSUE #665: Profiles service integration
**Product requirements/Acceptance criteria:**

Functional:
- [x] Get users from Profiles Service instead of DB 
- [ ] ~~Convert Journal JWT to xPub JWT~~
- [x] Remove OAuth

### PR DESCRIPTION
**Pull request by tamlyn**
#### Background

- Get user information from profiles service
- Use user and identities tables to store mapping between elife ID and PubSweet UUID
- Create login mock for development and e2e tests
   - Currently configured to log in with my eLife profile ID
- Remove auth-orcid component

#### Any relevant tickets

Closes #665 

#### How has this been tested?

Unit, integration and end to end tests updated and some new ones created

### CODE REVIEW

**Comment by diversemix** [comment]
@tamlyn ... is this dependant on https://github.com/elifesciences/elife-xpub/issues/637 before things will work?

File: config/default.js [hunk 1]
```diff
@@ -30,6 +30,9 @@ module.exports = {
     API_ENDPOINT: '/api',
     'login-redirect': '/',
   },
+  login: {
+    url: '', // TODO point this to Journal
```
Comment (jsms90) [inline config/default.js]: Yep @diversemix, this PR only gets things working in development/testing

Before things will work in demo/production, this line will need to be change as part of #637

File: server/xpub/entities/user/resolvers.js [hunk 2]
```diff
@@ -1,13 +1,21 @@
...
     async orcidDetails(_, vars, ctx) {
-      const user = await User.find(ctx.user)
-      return fetchOrcidDetails(user)
+      const user = await UserManager.findByProfileId(ctx.user)
+      const identity = user.identities[0]
+      return {
+        firstName: identity.meta.firstName,
+        lastName: identity.meta.lastName,
+        email: identity.email,
+        aff: identity.meta.affiliation,
```
Comment (diversemix) [inline server/xpub/entities/user/resolvers.js]: doesn't look like we are returning `meta.orcid` ... is there a reason for that?

File: server/xpub/entities/user/routes.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+const secret = config.get('pubsweet-server.secret')
+
+module.exports = app => {
+  if (config.get('login.enableMock')) {
+    app.get('/mock-token-exchange/:id', async (req, res) => {
+      const mockProfileId = req.params.id
+      logger.info('Signing mock JWT for profile ID', mockProfileId)
+      const xpubToken = await jwt.sign({ id: mockProfileId }, secret, {
+        expiresIn: '1y',
```
Comment (diversemix) [inline server/xpub/entities/user/routes.js]: Nice! :1st_place_medal:

**Pull request merged**